test/models/SessionsRepositorySpec.scala

```diff
@@ -25,7 +25,7 @@ class SessionsRepositorySpec extends PlaySpecification with Mockito {
   private val endOfDayMillis = endOfDayDate.getTime
```

Comment:
Would you want to change youtubeLink and slideShareLink to youtubeURL and slideShareURL in test cases as well?